xml-handler/src/resources/sweep/MP5000Sweep.xml

```diff
@@ -585,6 +585,11 @@
                                     step_source_complete_events)
                             end
 
+                            -- Adding step-to-sweep-delay    --
+                            if sweep_source_event ~= nil then
+                                tm_add.delay.constant(self.trigger_model, block.wait_sweep, stepToSweepDelay)

```

Comment:
You will need to add a new block name to the list above called something like `block.delay_sweep` and replace the reference to `block.wait_sweep`.